.gitignore

```diff
@@ -10,6 +10,6 @@
 
 /tests/Application/yarn.lock
 
-.env.test
-
```

Comment:
Is this change needed? This file is not part of this PR

Reply:
Yup, but since I changed something in `.gitignore` file, I wanted to fix and delete a file that shouldn't be there as we have env files only in test application directory